scripts/compressIni.sh

```diff
@@ -1,9 +1,9 @@
 # This script cleans up the ini file to remove comments and whitespace, so it only keeps the actual data
 # Remove lines that are solely comments with optional leading whitespace
-sed -i '/^[[:blank:]]*;/d' "$1"
+sed -i'' '/^[[:blank:]]*;/d' "$1"
```

Comment:
Looks like an extra space should be added here.

```
sed -i '' '/^[[:blank:]]*;/d' "$1"
```

<img width="695" alt="Screenshot 2023-06-01 at 15 15 59" src="https://github.com/jwplayer/ott-web-app/assets/27254986/4abd8c85-5b90-4470-9e5d-88ead60585af">

Reply:
Btw, should we also make it executable using `chmod`?

Reply:
@AntonLantukh Unfortunately with the space doesn't work for me on linux. I was hoping it would work on mac like that, but I guess not. Can you try the latest push?